src/Extractor/QueryFactory.php

```diff
@@ -30,11 +30,15 @@ protected function generate(): string
         $sql[] = 'FROM ' . $this->getFrom();
 
         if ($this->config->hasIncrementalFetchingKey()) {
-            if (isset($this->inputState['lastFetchedRow'])) {
+            if (isset($this->inputState[Config::STATE_LAST_FETCHED_ROW])) {
+                $lastFetchedRow = $this->inputState[Config::STATE_LAST_FETCHED_ROW];
+                if (!is_float($lastFetchedRow)) {
```

Comment:
Hm ... bolo by dobre vediet, ci tu nema byt aj is_int .... resp. namiesto `!is_float` => `is_string` ... skusal si to aj s int? ... `id` je v CosmosDb povinne string, takze na tom sa to neda vyskusat.

Reply:
jj ten is_string bude lepší 👍